### PR TITLE
[PAY-709][PAY-704][PAY-702][PAY-700][PAY-696][PAY-692][PAY-691] Various Stripe OnRamp Fixes pt 2

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

**Again, recommended to look commit by commit**

See commit names/linear items for details.

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx [hunk 1]
```diff
@@ -33,6 +40,9 @@ export const StripeBuyAudioButton = () => {
     purchaseInfo?.isError === false
       ? purchaseInfo.estimatedSOL.uiAmountString
       : undefined
+  const belowThreshold =
+    purchaseInfo?.isError === false && purchaseInfo.estimatedUSD.uiAmount < 1
+  console.log({ belowThreshold, purchaseInfo })
```
Comment (piazzatron) [inline packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx]: log

File: packages/web/src/components/buy-audio-modal/components/SuccessPage.tsx [hunk 2]
```diff
@@ -26,18 +27,23 @@ const messages = {
...
   const dispatch = useDispatch()
   const transactionDetails = useSelector(getTransactionDetails)
+  const onSuccessAction = useSelector(getOnSuccessAction)
   const [, setModalVisibility] = useModalState('BuyAudio')
   const [, setTransactionDetailsModalVisibility] =
     useModalState('TransactionDetails')
 
   const handleDoneClicked = useCallback(() => {
+    if (onSuccessAction) {
+      dispatch(onSuccessAction)
```
Comment (piazzatron) [inline packages/web/src/components/buy-audio-modal/components/SuccessPage.tsx:43]: should we also clear it here to be safe?

**Review by piazzatron — APPROVED** [review]
Looks great! Neat pattern with storing the on complete action in the slice

File: packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx (hunk 1 above)
Comment (piazzatron) [reply packages/web/src/components/buy-audio-modal/components/StripeBuyAudioButton.tsx]: edit: u removed it

File: packages/web/src/store/application/ui/buy-audio/sagas.ts [hunk 3]
```diff
@@ -90,7 +90,7 @@ const { getBuyAudioFlowStage, getFeesCache, getBuyAudioProvider } =
 const { increaseBalance } = walletActions
 const { fetchTransactionDetailsSucceeded } = transactionDetailsActions
 
-const SLIPPAGE = 3 // The slippage amount to allow for exchanges
+const DEFAULT_SLIPPAGE = 3 // The default slippage amount to allow for exchanges, overridden in optimizely
```
[the remark this answers is not in the document]
Comment (rickyrombo) [reply packages/web/src/store/application/ui/buy-audio/sagas.ts:93]: Yes unfortunately. It's pretty annoying but optimizely can in theory return null iirc

**Pull request merged**